x/oracle/types/keys.go

```diff
@@ -45,6 +46,18 @@ func GetOracleUpgradeKey(uniqueID string, address sdk.AccAddress) []byte {
 	return append(OracleUpgradeKey, CombineKeys([]byte(uniqueID), address)...)
 }
 
+func GetOracleUpgradeQueueKey(uniqueID string, addr sdk.AccAddress) []byte {
+	return append(OracleUpgradeQueueKey, CombineKeys([]byte(uniqueID), addr)...)
+}
```

Comment:
Since we can have only one `OracleUpgradeInfo`, how about using the key with combination of `OracleUpgradeQueueKey` and `addr` for oracle upgrade queue?

Reply:
That's a good point. Since the uniqueID of all elements int the queue will be the same , it seems better to remove it.